lms/djangoapps/course_goals/models.py

```diff
@@ -33,12 +35,16 @@ class Meta:
 
     user = models.ForeignKey(User, blank=False, on_delete=models.CASCADE)
     course_key = CourseKeyField(max_length=255, db_index=True)
+    # The goal a user has set for the number of days they want to learn per week
+    number_of_days_with_visits_per_week_goal = models.PositiveIntegerField(default=0)
```

Comment:
This variable/field name is maybe so verbose it wraps around to confusing again  :P  Do you like any of the following?
* Instead of `number_of_dates_with_visits_per_week`, maybe just `weekly_visits`?
* Instead of `_goal` at end, maybe `desired_`, `planned_`, or `expected_` at start, or `target` at either end? Maybe if the above suggestion is taken, this one becomes less relevant. But `_goal` isn't a big distinction in a class called `CourseGoal`.

Reply:
`number_of_dates_with_visits_per_week` doesn't still feel a little jarring to you? I think my favorite number of prepositions in a variable name is zero, followed closely by one. You didn't like `weekly_visits` I take it. What about `days_per_week`?

Reply:
sorry i read it quickly and thought you were including that as a suggestion
to me, number_of_dates_with_visits_per_week seems fine, but I'd also be ok with days_per_week_with_visits although i think that's a little worse

Reply:
I don't mean to be overly nit-picky here, but naming of columns is a little more important to get right the first time, over - say - a random variable name. And that long name still is a hard read to my eyes.

For example, the prefix `number_of_` feels like noise to me when I'm trying to parse the name. It's an int field, of course it's a number. And you counter-offered my `days_per_week` suggestion with a suffix of `_with_visits`, but visits feels implicit to me. But if you wanted to make it explicit, maybe simplify with like... `active_days_per_week` or something. The prepositions are what kill me. 

But maybe I'm diving too deep on this or it's just my brain -  try getting some another engineer's opinion on whether it's the name is confusing / if so, how to make it a little more parsable.

Reply:
updated to days_per_week per our slack conversationn